pkg/workflows/delete.go

```diff
@@ -21,13 +21,14 @@ type Delete struct {
 }
 
 func NewDelete(bootstrapper interfaces.Bootstrapper, provider providers.Provider,
-	clusterManager interfaces.ClusterManager, addonManager interfaces.AddonManager,
+	clusterManager interfaces.ClusterManager, addonManager interfaces.AddonManager, writer filewriter.FileWriter,
```

Comment:
What did the delete workflow need a `FileWriter` for?

Reply:
We will need it eventually if we decide to add a checkpoint  feature for that command but i can remove it for now